### ISSUE TITLE
Sidebar navigation is clipped or goes off-screen with longer content

### ISSUE THREAD

**Issue by jwflory** [unicef/inventory-hugo-theme#142]
# Summary

When viewing a category with several pages, the text in the sidebar will go off-screen when scrolling to the bottom of the page. This was only tested with larger viewports.

# How to reproduce?

1. Visit any of the example pages ([Software Toolkit](https://unicef.github.io/ooi-toolkit-software/technical-capabilities/version-control/), [DS Toolkit](https://unicef.github.io/ooi-toolkit-ds/04-data-collection-processing/missing-data.en.adoc/), [Drones Toolkit](https://unicef.github.io/drone-4sdgtoolkit/guides/humanitarian-drone-mission/))
2. Scroll to the bottom of the page.
3. Look at the sidebar. Some content will be clipped (i.e. flows off-screen) while other content blends into the footer.

# Expected behavior

The sidebar navigation is persistent on the viewport and the footer stops at the bottom of the sidebar navigation.

# Actual behavior

The sidebar navigation is overlayed on top of the footer and the sidebar content merges into the footer.

**Screenshots**:

![Screenshot from the Software Development Toolkit. The sidebar contents overflow into the footer. It is unclear whether longer sidebars would go off-screen entirely.](https://user-images.githubusercontent.com/4721034/178579326-5946315b-4c8e-441a-a2c7-725b4d7e2bb6.png "Screenshot from the Software Development Toolkit. The sidebar contents overflow into the footer. It is unclear whether longer sidebars would go off-screen entirely.")

![Screenshot from the Data Science Toolkit. The sidebar contents barely overflows out of the box for the sidebar. This is a more subtle example of the bug.](https://user-images.githubusercontent.com/4721034/178579366-ebe516bc-fea3-4850-a46b-568062ca18f7.png "Screenshot from the Data Science Toolkit. The sidebar contents barely overflows out of the box for the sidebar. This is a more subtle example of the bug.")

![Screenshot from the Drones Toolkit. The "Get PDF" button merges into the footer and the button loses its visible shape.](https://user-images.githubusercontent.com/4721034/178579384-3766874b-fd51-4295-bc4a-7cfeb94d86f2.png "Screenshot from the Drones Toolkit. The \"Get PDF\" button merges into the footer and the button loses its visible shape.")

**Comment by Neha9849** [comment]
@jwflory This one seems a bit more tricky than what I initially thought about it.
I thought of playing with the `z-index` property to fix this.
This would result in something like this-

Normal behavior:
![image](https://user-images.githubusercontent.com/85057583/178733395-9d55b384-467a-49e0-bad7-27bf722fecca.png)

When scrolled:
![image](https://user-images.githubusercontent.com/85057583/178733752-d2182f7b-f9d7-4d63-a096-b9072e444b5d.png)
 But the problem here is the sidebar won't be clickable as its `position` is` fixed`.
I thought of removing the position fixed and like this, the sidebar will also get scrolled with the content but this will fix this issue.
@jwflory What do you think?

**Comment by jwflory** [comment]
@Neha9849 It seems like this one will take more time to research on the right fix. I think the frequency of this issue is less, so we can deprioritize until after the DPG restructure is done. For now, let's keep this issue on the backlog. I also flagged it as a "good first issue" in case someone else has bandwidth to take it on before we add it to a sprint.

**Comment by jwflory** [comment]
Fixed by #150.

**Comment by jwflory** [comment]
It seems this regressed. Needs more investigation.

**Comment by jwflory** [comment]
This was actually fixed in #150. In Sprint 5 retrospective today, we realigned the `unicef/inventory-staging` `main` git branch to the same branch used for the Pull Request in unicef/inventory#136. This started working shortly after getting the git history to mirror the `unicef/inventory` base commit.

Closing this again as `completed` and grouping it back with Sprint 5 since it was actually fixed previously!
